jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -841,14 +832,13 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
 
         if include_pixarea_fac:
             # convert pixarea, which is in arcsec2/pix2 to the display solid angle unit / pix2
-
-            if self.config in ('imviz', 'deconfigged'):
+            if self.config == 'imviz':

```

Comment:
should this also be for images (not cubes) in deconfigged or no (because deconfigged has unit-conversion and imviz does not)?